scipy/sparse/linalg/isolve/lsmr.py

```diff
@@ -141,6 +141,7 @@ def lsmr(A, b, damp=0.0, atol=1e-6, btol=1e-6, conlim=1e8,
 
     Examples
     --------
+    >>> import numpy as np
```

Comment:
this is unnecessary, and we try to avoid adding it to every single example; the import is assumed.

Reply:
If I remember this right, I added it as for some reason PyCharm was complaining about it. But for other files not.